src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs

```diff
@@ -218,6 +230,72 @@ private List<string> GetExamples(XPathNavigator nav, ITripleSlashCommentParserCo
             return GetMultipleExampleNodes(nav, "/member/example").ToList();
         }
 
+        private void ResolveCodeSource(XDocument doc, ITripleSlashCommentParserContext context)
+        {
+            foreach (XElement node in doc.XPathSelectElements("//code"))
+            {
+                var source = node.Attribute("source");
+                if (source == null)
+                {
+                    continue;
+                }
+
+                var region = node.Attribute("region");
+
+                var path = source.Value;
+                if (!Path.IsPathRooted(path))
+                {
+                    path = Path.Combine(context.Source?.BasePath ?? ".", path);
```

Comment:
Good catch. Is the question what BasePath points at, or what the example source path should be relative to? I'll be honest, I'm not 100% sure what BasePath represents, but as is obvious from the unit test changes, I assumed it was the current comments' src file path. So, my intention was that the code would be relative to the current source.

In Sandcastle, this wasn't the case. The path was relative to the SHFB project. I always found that confusing, and since there's little chance of existing code not having to update these paths regardless, I thought I'd do what made sense to me. That is, assuming BasePath meant what I thought it did. If it didn't, then this behavior is probably closer to what Sandcastle did. I can change this part of the code if anyone thinks it needs to be different, though. Just let me know what it should be instead.